src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx

```diff
@@ -47,9 +46,9 @@ export default function PageFooter(props: Props) {
     }
   }
 
-  // Handler to update ShouldBeInLocal state when the checkbox is toggled
-  const onCheckboxChange = (e: CheckboxChangeEvent) => {
-    dispatch(setShouldBeInLocal(e.target.checked));
+  // Handler to update ShouldBeInLocal state when the switch is toggled
+  const onSwitchChange = (checked: boolean) => {
+    dispatch(setShouldBeInLocal(checked));
```

Comment:
I think `shouldBeInLocal` should be equivalent to`!checked`--if the switch is 'on' (`checked=true`, I assume), it's a cloud-only upload, so `shouldBeInLocal` should be `false`

Reply:
resolved in [8e6f957](https://github.com/aics-int/aics-file-upload-app/pull/120/commits/8e6f957f0e6697814efd10e9f804a8d71ba4c5dc)